packages/web/src/scss/components/Icon/_Icon.scss

```
@@ -0,0 +1,18 @@
+@use '@tokens' as tokens;
+@use '../../tools/responsive-properties';
+@use 'theme';
+
+svg {
+    @include responsive-properties.create-cascade(
+        $property: 'width',
+        $input-custom-property-base-name: '#{tokens.$css-variable-prefix}icon-width',
+        $breakpoints: theme.$breakpoints,
+        $fallback-value: 'fit-content'
+    );
+    @include responsive-properties.create-cascade(
+        $property: 'height',
+        $input-custom-property-base-name: '#{tokens.$css-variable-prefix}icon-height',
+        $breakpoints: theme.$breakpoints,
+        $fallback-value: 'fit-content'
+    );
```

As @crishpeen suggests, this can be a bit simpler:

```suggestion
    @include responsive-properties.create-cascade(
        $property: 'width',
        $input-custom-property-base-name: '#{tokens.$css-variable-prefix}icon-size',
        $breakpoints: theme.$breakpoints,
        $fallback-value: 'fit-content'
    );
    @include responsive-properties.create-cascade(
        $property: 'height',
        $input-custom-property-base-name: '#{tokens.$css-variable-prefix}icon-size',
        $breakpoints: theme.$breakpoints,
        $fallback-value: 'fit-content'
    );
```